source/server/admin/prometheus_stats.cc

```diff
@@ -16,6 +16,18 @@ const Regex::CompiledGoogleReMatcher& promRegex() {
   CONSTRUCT_ON_FIRST_USE(Regex::CompiledGoogleReMatcher, "[^a-zA-Z0-9_]", false);
 }
 
+const Regex::CompiledGoogleReMatcher& backslashRegex() {
+  CONSTRUCT_ON_FIRST_USE(Regex::CompiledGoogleReMatcher, R"(\\)", false);
```

Comment:
Regular expressions are overkill for what you're doing here. I think `absl::StrReplaceAll` will do what you need and be simpler.

Reply:
thanks for the pointer. I was not aware of `StrReplaceAll` (somehow missed in a quick skim of docs). Updated.